app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -3,7 +3,7 @@
 module UspsInPersonProofing
   class EnrollmentHelper
     class << self
-      def schedule_in_person_enrollment(user, pii, opt_in = nil)
+      def schedule_in_person_enrollment(user, pii, is_enhanced_ipp, opt_in = nil)
```

Comment:
IMO this is a good candidate for a  keywords arguments reactor. 4 positional arguments, the last one optional, and the last 2 both booleans is ripe for confusion

Reply:
@zachmargolis See [9d3bf7a](https://github.com/18F/identity-idp/pull/10820/commits/9d3bf7a8193b09869cfaec29d7466a7e79dd26b5).

Reply:
big fan! thx